### PR TITLE
[YUNIKORN-1105] Use bytes as base unit for memory

### PR DESCRIPTION
**Pull request by craigcondit**
### What is this PR for?
Use bytes as the base unit of memory, as the scheduler will now use absolute values.

This should be committed at the same time as the shim PR.

### What type of PR is it?
* [ ] - Bug Fix
* [x] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-1105

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

**Review by wilfred-s — APPROVED** [review]
LGTM

**Comment by wilfred-s** [comment]
@akhilpb001 would like to make sure that you have seen this change before we commit

File: src/app/utils/common.util.ts [hunk 1]
```diff
@@ -30,9 +30,19 @@ export class CommonUtil {
   }
 
   static formatMemory(value: number | string): string {
-    let unit = 'MB';
+    let unit = 'bytes';
```
Comment (akhilpb001) [inline src/app/utils/common.util.ts:33]: nit: Would it be better to the casing of `bytes` to `Bytes`?

**Review by akhilpb001 — APPROVED** [review]
LGTM except minor nit.

**Pull request closed without merging**